### ISSUE TITLE
kerl exits upon installation preventing multiple installs

### ISSUE THREAD

**Issue by deybhayden** [asdf-vm/asdf-erlang#61]
Hey there, I think this is what I'm experiencing, but given this tool-versions file:

```
erlang 20.0
erlang 20.2.2
elixir 1.5.1
elixir 1.6.1
nodejs 8.5.0
nodejs 8.9.4
postgres 9.6.5
postgres 9.6.6
```

And running this command

```
$> asdf install
```

The process exits after installing the first erlang version. I think I started experience this behavior after the repo was updated to kerl.

**Comment by febeling** [comment]
Duplicate of #57.

**Comment by Stratus3D** [comment]
Follow #57 for the fix.

**Comment by febeling** [comment]
There isn’t a description of a fix in that issue #57, afaict.

**Comment by Stratus3D** [comment]
Sorry, my comment was misleading. There is not presently a fix, but when this issue is address it will be fixed under #57.

I think we are going to see which kerl command is exiting with a non-zero exit code and use something like `<kerl command> || true` to make sure the plugin script exits with a status of 0.
